src/app/api/admin/SessionID/route.ts

```diff
@@ -50,12 +51,13 @@ export async function POST(req: NextRequest)
         response.cookies.set('sessionID', sessionID, {
             httpOnly: true,
             // secure: true,
-			sameSite: 'none',
+			sameSite: 'strict',
```

Comment:
browsers reject SameSite: None, when it's not Secure

https://github.com/42KL-admin/42DoGood-Reef_Check/assets/20178082/ad844076-e2ec-4744-bff5-af3e0b1844a9